outputs.tf

```diff
@@ -22,3 +22,8 @@ output "vpc_id" {
   description = "AWS identifier for the VPC provisioned by this module"
   value       = module.aws-vpc.vpc_id
 }
+
+output "igw_id" {
```

Comment:
I think it would be helpful to add `igw_arn` as well just in case the name isn't unique?

Reply:
We just added it